src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UITests/Presentation/SampleChooserViewModel.cs

```diff
@@ -278,6 +280,7 @@ internal async Task RecordAllTests(CancellationToken ct, string screenShotPath =
 					async () =>
 					{
 						await RecordAllTestsInner(folderName, ct, doneAction);
+						_isRecordAllTests = false;
```

Comment:
It may be interesting to put it in a try/finally, in case the Record fails unexpectedly:
```suggestion
						try {
						  await RecordAllTestsInner(folderName, ct, doneAction);
						}
						finally {
						  _isRecordAllTests = false;
						}
```